AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -84,6 +84,9 @@ motorcyclenews.com,finanzen.net,vorname.com,jezebel.com,theonion.com,theinventor
 /wrapperMessagingWithoutDetection.*.js$script,domain=netzwelt.de
 !
 !
+||mangaplus.shueisha.co.jp/img/img_gdpr_
+mangaplus.shueisha.co.jp##img[src^="/img/img_gdpr_"] + div[class^="styles-module_body_"]
```

Comment:
I guess it's ok. Also you can try `mangaplus.shueisha.co.jp###app > div.vue-notification-group + div > div[class^="styles-module_container_"]:first-child` instead of them. Please check it on your end.

Reply:
Actually I also thought about that, but `first-child` may be vulnerable to furure change of the site.

Reply:
Tbh, it's fair for every rule, but ok.